src/components/BookItem/_BookItem.scss

```diff
@@ -72,16 +72,17 @@
 
 .delete-book {
   .delete-confirm-btn {
-    border-radius: 4px;
+    font-weight: bold;
+    border-radius: 6px;
     height: 30px;
     border: none;
     color: white;
     font-size: 14px;
     position: absolute;
-    top: $padding-xs;
-    right: $padding-xs;
```

Comment:
Because this $padding-xs variable is not applicable everywhere.  And I don't agree that all specifications should be defined in a variable since they are  not reusable and the usage of variable in these cases would only introduce ambiguitt